db/seeds.rb

```diff
@@ -1 +1,3 @@
 AdminUser.create!(email: 'admin@example.com', password: 'password')
+
+User.create!(github_name: 'dependabot', blacklisted: true)
```

Comment:
So, if the user is blacklisted, we are not sending the PR to slack?

Reply:
yes, that is what the new test at spec/services/slack_notification_service_spec.rb in #89 validates.